lms/djangoapps/ora_staff_grader/views.py

```diff
@@ -147,6 +150,121 @@ def get(self, request, ora_location, *args, **kwargs):
             return UnknownErrorResponse()
 
 
+class AssessmentFeedbackToView(StaffGraderBaseView):
```

Comment:
FYI, it is possible to declare a single DRF view with multiple "GET" endpoints: https://stackoverflow.com/a/70301277

Reply:
Hi @pomegranited, I didn't know that was possible, thanks for commenting! I have already made the necessary changes to use a single view. Could you check it again?